lib/media/ttml_text_parser.js

```diff
@@ -332,12 +343,15 @@ shaka.media.TtmlTextParser.addStyle_ = function(
       cueElement, region, styles, 'tts:textAlign');
   if (align) {
     cue.align = align;
-    if (align == 'center' && cue.align != 'center') {
-      // Workaround for a Chrome bug http://crbug.com/663797
-      // Chrome does not support align = 'center'
```

Comment:
I'm a little lost here. Did something change in terms of logic on lines 335-351?

I suspect you might be working off an older version of the file (we refactored it a bit since your last commit). If that's the case, let's stick to the newer version.

Looks good otherwise.